R/plotPercentiles.R

```diff
@@ -0,0 +1,108 @@
+#' Comparison plots show 50th percentile of user selected variables as obtained from different scenario runs. If
+#' available in the data, ribbon plots will also show the 33th - 67th percentile in a darker color and the 5th – 95th
```

Comment:
Here you write "if available" (as in: optional), below you say "must be given" (as in: mandatory).  Can you clarify?